_slides/16-mdjaniga.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+No Clue What I Am!
```

Comment:
Clearly this is a gold mining octo cat.  I don't have any better idea for a caption though.

Reply:
"Eureka!"???